test/build.sh

```diff
@@ -22,8 +22,8 @@ set -o nounset
 set -o pipefail
 
 
-NPD_STAGING_PATH=${NPD_STAGING_PATH:-"gs://node-problem-detector-staging"}
-NPD_STAGING_REGISTRY=${NPD_STAGING_REGISTRY:-"gcr.io/node-problem-detector-staging"}
+NPD_STAGING_PATH=${NPD_STAGING_PATH:-"gs://k8s-staging-npd"}
```

Comment:
we should fetch from github, not GCS

nobody should be fetching from GCS, it's not cost effective. sometimes we will have GCS behind a CDN, or better yet use github's infra

Reply:
oh I see, this is only for testing the project itself, that's OK.